### PR TITLE
AiSD-9-quickSort

### PR DESCRIPTION
**Pull request by mztezi**

### CODE REVIEW

File: AiSD-9-quickSort.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    swap(arr, i, high);
+    return i;
+}
+
+function quickSort(arr, low = 0, high = arr.length - 1) {
+    if (low >= high || low < 0) {
+        return;
+    }
+
+    let pi = partition(arr, low, high);
```
Comment (kdryzek) [inline AiSD-9-quickSort.js]: it's not pi

Comment (kdryzek) [reply AiSD-9-quickSort.js]: and not let

File: AiSD-9-quickSort.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    for (let j = low; j < high; j++) {
+        if (arr[j] < pivot) {
+            i++;
+            swap(arr, i, j);
+        }
+    }
+
+    i++;
+    swap(arr, i, high);
+    return i;
```
Comment (kdryzek) [inline AiSD-9-quickSort.js]: empty line above return

Comment (mztezi) [reply AiSD-9-quickSort.js]: fixed

File: AiSD-9-quickSort.js (hunk 1 above)
Comment (mztezi) [reply AiSD-9-quickSort.js]: fixed

**Review by kdryzek — APPROVED** [review]

**Pull request merged**